### ISSUE TITLE
tped rs_ids not outputted

### ISSUE THREAD

**Issue by davemcg** [arq5x/gemini#786]
Issue in 0.19.0

The gemini variant_id is being printed instead of the rs_id when you run gemini query --format tped. I think the issue is on line 191 for this script:
https://github.com/arq5x/gemini/blob/d2f8d70e942643bb7327df3c84e4392692730750/gemini/GeminiQuery.py

```
name = str(row['variant_id'])
return " ".join([chrom, name, "0", start, genotypes])
```
I think 'variant_id' is supposed to be 'rs_ids'

**Comment by brentp** [comment]
good catch. I'll get this in for the next release after I merge the py3 branch.
